codemirror.tera

```diff
@@ -0,0 +1,117 @@
+---
+accent: mauve
+whiskers:
+  version: "2.3.0"
+  matrix:
+    - flavor
+  filename: "themes/{{ flavor.identifier }}.css"
+---
+div.cm-s-ctp-{{ flavor.identifier }}.CodeMirror {
+  --bg0: #{{ base.hex }};
+  --bg1: #{{ overlay1.hex }};
+  --bg4: #{{ overlay0.hex }};
+  --fg: #{{ text.hex }};
+  --fg3: #{{ subtext0.hex }};
+  --gray: #{{ subtext1.hex }};
+  --blue: #{{ blue.hex }};
+  --yellow: #{{ yellow.hex }};
+  --aqua: #{{ blue.hex }};
+  --orange: #{{ peach.hex }};
+  --primary-bg: #{{ base.hex }};
+  --current-line: #{{ surface0.hex }};
+  --selection: #{{ surface2.hex }};
+  --atom: #{{ lavender.hex }};
+  --cursor: #{{ overlay1.hex }};
+  --keyword: #{{ red.hex }};
+  --operator: #{{ sky.hex }};
+  --number: #{{ peach.hex}};
+  --definition: #{{ blue.hex }};
+  --string: #{{ green.hex }};
+}
+
+.cm-s-ctp-{{ flavor.identifier }}.CodeMirror,
```

Comment:
```suggestion
.cm-s-ctp-{{ flavor.identifier }}.CodeMirror,
```

This line is super repetitive. I would set it to a variable like `{% set prefix = ".cm-s-ctp-" ~ flavor.identifier %}`.